Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/tangential_relaxation.h

```diff
@@ -115,46 +116,64 @@ struct Allow_all_moves{
 *     \cgalParamDefault{If not provided, all moves are allowed.}
 *   \cgalParamNEnd
 *
+*   \cgalParamNBegin{sizing_function}
+*     \cgalParamDescription{An object containing sizing field for individual vertices.
+*                           Used to derive smoothing weights.}
+*     \cgalParamType{A model of `PMPSizingField`.}
+*     \cgalParamDefault{If not provided, smoothing weights are the same for all vertices.}
+*   \cgalParamNEnd
+*
 * \cgalNamedParamsEnd
 *
 * \todo check if it should really be a triangle mesh or if a polygon mesh is fine
 */
-template <typename VertexRange, class TriangleMesh, class NamedParameters = parameters::Default_named_parameters>
+template <typename VertexRange,
+          class TriangleMesh,
+          typename CGAL_NP_TEMPLATE_PARAMETERS>
```

Comment:
this will not appear as expected in the documentation. Is there an issue with `NamedParameters`?

Reply:
Put it there during my attempts to combine tangential relaxations. Will try to revert it.

Reply:
Don't revert anything, this is correct. The macro is expended by doxygen.